__tests__/app.test.js

```diff
@@ -45,4 +45,28 @@ describe('app.js', () => {
         });
     });
   });
+  describe('/api/reviews', () => {
+    test('returns a list of reviews in an object key with array values', () => {
+      return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then((response) => {
+          console.log(response.body);
+          expect(typeof response.body).toBe('object');
+          expect(response.body.reviews.length).toBeGreaterThan(0);
+          expect(
+            response.body.reviews.forEach((review) => {
+              expect(review).toEqual(
+                expect.objectContaining({
+                  review_id: expect.any(Number),
+                  title: expect.any(String),
+                  category: expect.any(String),
+                  votes: expect.any(Number),
+                })
```

Comment:
This test is missing a whole bunch of the keys we're expecting, according to the ticket.